src/components/todo/Todo.tsx

```diff
@@ -1,97 +1,78 @@
-//       )
-//     );
-//   };
+function TodoList() {
+  const { filteredTasks, handleCheckboxChange, dispatch } = useTodoContext();
 
-//   const selectAll = () => {
-//     const allSelected = tasks.every(task => task.completed);
-//     setTasks(tasks.map(task => ({ ...task, completed: !allSelected })));
-//   };
+  return (
+    <div>
+      {filteredTasks.map((task: any, index: number) => (
+        <div className="check" key={task.task + index}>
+          <input
+            type="checkbox"
+            checked={task.completed}
+            onChange={() => handleCheckboxChange(index)}
+          />
+          <label>{task.task}</label>
+          <button onClick={() => dispatch({ type: "delete-task", payload: index })}>
+            X
+          </button>
+        </div>
+      ))}
+    </div>
+  );
+};
 
-//   const sortedTasks = tasks.filter(task => {
-//     if (filtered === "Active") {
-//       return !task.completed;
-//     }
-//     if (filtered === "Completed") {
-//       return task.completed;
-//     }
-//     return true;
-//   });
-//   return (
-//     <div className="App">
-//       <button type="button" className="btn select" onClick={selectAll}>
-//         {tasks.every(task => task.completed) ? "Deselect All" : "Select All"}
-//       </button>
-//       <input
-//         type="text"
-//         value={input}
-//         onChange={handleInputChange}
-//         onKeyDown={handleKeyPress}
-//         placeholder="What needs to be done?"
-//         className="head"
-//       />
+function TodoFooter() {
+  const { taskData, dispatch } = useTodoContext();
 
-//       <div className="body">
-//         {sortedTasks.map((task, index) => (
-//           <div className="check" key={index}>
-//             <input
-//               type="checkbox"
-//               checked={task.completed}
-//               onChange={() => handleCheckboxChange(index)}
-//             />
-//             <label>
-//               {task.task}
-//             </label>
-//             <button onClick={() => setTasks(tasks.filter((item, idx) => idx !== index))}>X</button>
-//           </div>
-//         ))}
-//       </div>
-
-//       <button className="btn left">{`${tasks.length} items left!`}</button>
-//       <button className="btn all" onClick={() =>setFiltered("All")}>All</button>
-//       <button className="btn active" onClick={() =>setFiltered("Active")}>Active</button>
-//       <button className="btn completed" onClick={() =>setFiltered("Completed")}>Completed</button>
-//       <button
-//         className="btn clear"
-//         onClick={() => {
-//           const newTasks = tasks.filter(task => !task.completed);
-//           setTasks(newTasks);
-//         }}
-//       >
-//         Clear completed
-//       </button>
-//     </div>
-//   );
-// }
-
-// export default Todo;
\ No newline at end of file
+  return (
+    <div>
+      <button className="btn left">{`${taskData.tasks.length} items left!`}</button>
```

Comment:
this is not a button. this should be information of active tasks. We can use div and give border.